### ISSUE TITLE
gifs can no-longer be used as thumbnails

### ISSUE THREAD

**Issue by CuddleBear92** [game-list/game-list#136]
Figured out why some titles wont get a new thumbnail as you refresh the screenshot gallery, they are animated gifs. Or atleast a HUGE part of them is.

Animated gifs used to work fine as thumbnails, they where static in the gallery view but they did failed animations in the details page.
Personally would like them to animate in both the gallery and details page and animated correctly.... aslong as its performant! 

related to #129 #133 #92

**Comment by game-list** [comment]
Yes, I've seen that, I'm working on a way to update games because a lot of them don't have tags. 

As soon as I have finished that, I'll work on the thumbs.
